src/OpenSpaceToolkit/Astrodynamics/Dynamics/Tabulated.cpp

```diff
@@ -117,14 +125,14 @@ Array<Shared<const CoordinateSubset>> Tabulated::getReadCoordinateSubsets() cons
 
 Array<Shared<const CoordinateSubset>> Tabulated::getWriteCoordinateSubsets() const
 {
-    return writeCoordinateSubsets_;
+    return writeCoordinateSubset_;
```

Comment:
```suggestion
    return writeCoordinateSubsets_;
```
This is plural as we are returning subsets, like the function name says.
I apologize if the refactor was unclear but here are semantics:
CoordinateSubset -> A single Coordinate Subset
CoordinateSubsets -> an Array with each element of type Coordinate Subset